shared/selene/data/wake_word/repository/sql/add_wake_word.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO
```

Comment:
I am missing something here as this confuses me. How are the other wake word columns inserted?

Reply:
I leave out columns that have default values and the defaults are used.